pkgs/development/python-modules/python-prctl/default.nix

```diff
@@ -13,9 +14,21 @@ buildPythonPackage rec {
     sha256 = "b4ca9a25a7d4f1ace4fffd1f3a2e64ef5208fe05f929f3edd5e27081ca7e67ce";
   };
 
-  patches = [ ./skip_bad_tests.patch ];
   buildInputs = [ libcap ];
 
+  checkInputs = [ pytestCheckHook ];
+
+  disabledTests = [
+    # The Nix build sandbox has no_new_privs already enabled
+    "test_no_new_privs"
+
+    # The Nix build sandbox has seccomp already enabled
+    "test_seccomp"
+
+    # The Nix build sandbox has speculation_ctrl already enabled
+    "test_speculation_ctrl"
```

Comment:
Not exactly; it's not specific to the Nix build sandbox. The comment should be something like:
```
# This will fail if prctl(PR_SET_SPECULATION_CTRL, PR_SPEC_FORCE_DISABLE) has been set system-wide, even outside the sandbox
```